packages/graphql/tests/integration/math.int.test.ts

```diff
@@ -0,0 +1,809 @@
+            });
+
+            expect(gqlResult.errors).toBeUndefined(); 
+            const storedValue = await session.run(
+                `
+                MATCH(b: Actor{id: $id}) -[c: ACTED_IN]-> (a: Movie) RETURN c.pay as pay
+                `,
+                {
+                    id
+                }
+            );
+            expect(storedValue.records[0].get('pay')).toEqual(initialPay + payIncrement);
+
+        } finally {
+            await session.close();
+        }
+      });
+
+      test('Should raise in case of ambigous properties on relationships', async () => {
+        const session = driver.session();
+        const initialPay = 100;
+        const payIncrement = 50;
+        const typeDefs = `
+        type Movie {
+            title: String
+            viewers: Int
+            actors: [Actor!]! @relationship(type: "ACTED_IN", properties: "ActedIn", direction: IN)
+        }
+        
+        type Actor {
+            id: ID!
+            name: String!
+            actedIn: [Movie!]! @relationship(type: "ACTED_IN", properties: "ActedIn", direction: OUT)
+        }
+
+        interface ActedIn @relationshipProperties {
+            pay: Float
+        }
+        `;
+
+        const neoSchema = new Neo4jGraphQL({ typeDefs });
+
+        const id = generate({
+            charset: "alphabetic",
+        });
+
+        const query = `
+        mutation Mutation($id: ID, $payIncrement: Float) {
+            updateActors(where: { id: $id }, update: {
+                  actedIn: [
+                    {
+                      update: {
+                        edge: {
+                          pay_ADD: $payIncrement
+                          pay: $payIncrement
+                        }
+                      }
+                    }
+                  ]
+                }) {
+              actors {
+                name
+                actedIn {
+                  title
+                }
+                actedInConnection {
+                  edges {
+                    pay
+                  }
+                }
+              }
+            }
+        }
+
+        `;
+
+        try {
+            // Create new movie
+            await session.run(
+                `
+                CREATE (a:Movie {title: "The Matrix"}), (b:Actor {id: $id, name: "Keanu"}) WITH a,b CREATE (a)<-[actedIn: ACTED_IN{ pay: $initialPay }]-(b) RETURN a, actedIn, b
+                `,
+                {
+                    id,
+                    initialPay
+                }
+            );
+            // Update movie
+             const gqlResult = await graphql({
+                schema: await neoSchema.getSchema(),
+                source: query,
+                variableValues: { id, payIncrement },
+                contextValue: { driver, driverConfig: { bookmarks: session.lastBookmark() } },
+            });
+
+            expect(gqlResult.errors).toBeDefined(); 
```

Comment:
You can consider adding a check for the specific error message here.

Reply:
I don't add a specific error type for some errors, oveflows for example, a Java Long overflow error is raised before the apoc.validate could raise his own, in this way I hope I could keep the feature less dependent on the cypher/Neo4j implementation behind :)

Reply:
We are pretty depended on the cypher and Neo4j implementation in general. We could see it the other way around, if the cypher or Neo4j implementation changes then we have it covered with our tests which we can update quite easily. It would also flag us about changes in cypher and Neo4j.
I'm okay however not to include details.

Reply:
I just notice that in this case specific there is not a reason at all to not specify which error is raised, as the error that we expect in this case is the one owned by us! But I think I'm going to match the specific error also the others cases 👍